migrate.py

```diff
@@ -188,17 +188,17 @@ def visit_Assign(self, node):
     return plugin_data, deps
 
 
-def rewrite_imports(mod_src_text, coll, spec, namespace):
+def rewrite_imports(mod_src_text, collection, spec, namespace):
     """Rewrite imports map."""
-    plugins_path = ('ansible_collections', namespace, coll, 'plugins')
```

Comment:
Why are you removing `ansible_collections`?

Reply:
Sorry that was an oversight.